CspSubmissionPlugin.inc.php

```diff
@@ -711,57 +707,22 @@ function TemplateManager_fetch($hookName, $args) {
 
 			return true;
 		}elseif ($args[1] == 'controllers/grid/users/stageParticipant/addParticipantForm.tpl') {
-			if($stageId == 5){
-				$locale = AppLocale::getLocale();
-				$userDao = DAORegistry::getDAO('UserDAO');
-				$result = $userDao->retrieve(
-					<<<QUERY
-					SELECT t.email_key, o.body, o.subject
-					FROM email_templates t
-					LEFT JOIN
-					(
-						SELECT a.body, b.subject, a.email_id
-						FROM
-						(
-							SELECT setting_value as body, email_id
-							FROM ojs.email_templates_settings
-							WHERE setting_name = 'body' AND locale = '$locale'
-						)a
-						LEFT JOIN
-						(
-								SELECT setting_value as subject, email_id
-								FROM ojs.email_templates_settings
-								WHERE setting_name = 'subject' AND locale = '$locale'
-						)b
-						ON a.email_id = b.email_id
-					) o
-					ON o.email_id = t.email_id
-					WHERE t.enabled = 1 AND t.email_key LIKE 'LAYOUT%'
-					QUERY
-				);
-				$i = 0;
-				while (!$result->EOF) {
-					$i++;
-					$templateSubject[$result->GetRowAssoc(0)['email_key']] = $result->GetRowAssoc(0)['subject'];
-					$templateBody[$result->GetRowAssoc(0)['email_key']] = $result->GetRowAssoc(0)['body'];
 
-					$result->MoveNext();
-				}
+			if($stageId == 3 OR $stageId == 1){
 
-				$templateMgr = TemplateManager::getManager($request);
-				$templateMgr->assign(array(
-					'templates' => $templateSubject,
-					//'stageId' => $stageId,
-					//'submissionId' => $this->_submissionId,
-					//'itemId' => $this->_itemId,
-					'message' => json_encode($templateBody),
-					'comment' => reset($templateBody)
-				));
+				$mail = new MailTemplate('EDITOR_ASSIGN');
+				$templateSubject['EDITOR_ASSIGN'] = $mail->_data["subject"];
+				$templateBody['EDITOR_ASSIGN'] = $mail->_data["body"];
```

Comment:
essas variaveis foram declaras antes em algum lugar? se não, eu mudaria pra algo assim:
```suggestion
				$templateBody = ['EDITOR_ASSIGN' => $mail->_data["body"]];
```